10up/ElasticPress
# Fix WP_Query_User when it tries to sort the results

### Description of the Change

This changes updates to use keyword fields to sort in the WP_User_Query and adds tests to ensure we are using these fields.

### Alternate Designs

### Benefits

Elasticsearch will not fail and fall back to MySQL.

### Possible Drawbacks

### Verification Process

1. Enable Users feature
2. Go to a page/post and add a paragraph block
3. Type `@` and try to search a user e.g. `@admin`
4. Check if the results were provided by Elasticsearch.

### Checklist:






### Applicable Issues

Closes: #2225 

### Changelog Entry

Fixed sort for WP_User_Query
